wsd/AdminModel.cpp

```diff
@@ -895,6 +897,24 @@ int AdminModel::getPidsFromProcName(const std::regex& procNameRegEx, std::vector
     return pidCount;
 }
 
+int AdminModel::getAssignedKitPids(std::vector<int> *pids)
+{
+    return getPidsFromProcName(std::regex("kitbroker_.*"), pids);
```

Comment:
The string `kitbroker` and `kit_spare` should come from some common place, with a named constant. Common.hpp is a reasonable place to have it there. Perhaps we can do that later.

Reply:
Agree.